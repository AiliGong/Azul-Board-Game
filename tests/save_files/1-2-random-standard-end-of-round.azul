#myformat
3
5
0
a
b
LULRBRUURRYLRYLLYBUBUYUYYURYLYRBULLBBBUYRYRLLLUYRYYBLLYRBLRBUYUURRBRUBLUURBBBRBLYYBYULLUBRLRUYBBLYUR
turn 1 L 2
turn 2 U 2
turn 3 Y 1
turn 4 L 3
turn 0 R 5
turn 5 U 1
turn 0 B 3
turn 0 L 3
turn 0 Y 4
turn 0 U 4
turn 1 Y 4
turn 2 Y 2
turn 3 L 1
turn 4 B 1
turn 5 B 2
turn 0 U 5
turn 0 L 5
turn 0 R 3
turn 0 Y 3
turn 0 B 0
