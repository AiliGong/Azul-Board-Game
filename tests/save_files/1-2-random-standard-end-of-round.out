Factories:
0: F
1: R Y R L
2: L L U Y
3: R Y Y B
4: L L Y R
5: B L R B

Score for a: 7
Mosaic for a:
1:           . || . Y . . L . 
2:         . . || . B . . . L 
3:       . Y Y || . . B . . . 
4:     . . . . || . . . . Y . 
5:   . . . L L || R . . . . . 
6: . . . . . . || . . . . . . 
broken:

Score for b: 4
Mosaic for b:
1:           . || B . . U . . 
2:         . . || . . Y . U . 
3:       . R R || L . . . . . 
4:     . . . U || . . . . . . 
5:   . . . . . || . U . . . . 
6: . . . . . . || . . . . . . 
broken:
